### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by cbernat**
Updated readme with the last talk we gave about this.

### CODE REVIEW

**Comment by IgnacioRodrigues** [comment]
Good Ceci, it's okey for me.  👍

File: README.md [hunk 1]
```diff
@@ -1,13 +1,36 @@
 # Doppler-UI-System
-UI guide system
+UI live guide style. This library is meant to be Doppler visual languaje, to centralize all branding in one place for all our products. Also allowing to decouple UI from Front End logic, and work in a more tidy way. 
 
-# Instructions
+# What elements do we have right now on this library?
```
Comment (andresmoschini) [inline README.md]:
```suggestion
# What elements do we have right now on this library?

```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+Please check the cdn: http://cdn.fromdoppler.com/doppler-ui-library/latest/index.html
+
+# How can you use this library in you own project?
```
Comment (andresmoschini) [inline README.md]: En Markdown siempre va una interlinea entre párrafos.

```suggestion
# How can you use this library in you own project?

```

**Comment by GusBaamonde** [comment]
Excellent!! :)

**Pull request merged**